src/Database/Schema/PostgresSchema.php

```diff
@@ -382,8 +389,10 @@ public function columnSql(TableSchema $schema, string $name): string
             TableSchema::TYPE_TIME => ' TIME',
             TableSchema::TYPE_DATETIME => ' TIMESTAMP',
             TableSchema::TYPE_DATETIME_FRACTIONAL => ' TIMESTAMP',
+            TableSchema::TYPE_DATETIME_TIMEZONE => ' TIMESTAMPTZ',
```

Comment:
Should there be two abstract type if they share a concrete type in the only database that supports timezones right now? I think having a single type is ok.

Reply:
In postgres, `TIMESTAMPTZ` is an alias for `TIMESTAMP WITH TIME ZONE`.  It lets you do `TIMESTAMPTZ(3)` instead of `TIMESTAMP(3) WITH TIME ZONE`.  They keep the long name for compatibility with the SQL Standard names.

Reply:
Oh, nevermind, you're asking about `TYPE_DATETIME_TIMEZONE`.  There is one type that would use it: `datetimeoffset` in SqlServer.

https://docs.microsoft.com/en-us/sql/t-sql/data-types/datetimeoffset-transact-sql?view=sql-server-ver15

I added it for completion, but you're correct no concrete types map to it right now.